codebuild/spec/buildspec_musl.yml

```diff
@@ -23,7 +23,8 @@ phases:
     on-failure: ABORT
     commands:
       # Install musl libc
-      - git clone https://git.musl-libc.org/git/musl $MUSL_DIR
+      - aws s3 cp s3://s2n-public-test-dependencies/musl_repo_20240610.tgz .
```

Comment:
Rude of us to include a time-bomb, but I think that's probably healthier. Could we include some date check such that this will fail if we haven't updated the s3 archive for more than a quarter (90 days?)

Reply:
90 days seems agressive, but agreed- we have a number of pinned versions we need to keep better track of, issue #4591